### DISCUSSION TITLE
Developments in other libraries.

### DISCUSSION THREAD

**Discussion by quimt** [HapticX/happyx#109] [General]
FYI, 

I made a PR on nimib to support `nbHappyxCode`  blocks along with Karax. This will not only make it convenient to do some direct interactive comparisons between the DSLs, but it will let happyX be used in the nimib ecosystem generally. In particular, using **nimislides**, one will be able to make slideshows with interactive happyX apps in them.

**Comment by Ethosa** [comment]
sounds great!

**Comment by pietroppeter** [comment]
thanks for doing that @quimt! PR was merged the other day so it is available in `nimib#head`, we will soon (TM) make a release with this and other changes. Looking forward to play with Happyx!
